simplq/src/components/common/Modal/Modal.module.scss

```diff
@@ -0,0 +1,22 @@
+@import 'components/common.scss';
+
+.root {
+  overflow: scroll;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+}
+
+.content {
+  position: absolute;
+  top: 20%;
+  background-color: #fff;
+  border-radius: 25px;
+  outline: none;
+  width: 400px;
+  padding: 10px;
+  @media (max-width: 960px) {
+    width: 95%;
+    margin: 10px;
+  }

```

Comment:
One more tiny thing, I missed this in my last review, could you convert the sizing to use rem instead of px. We're using rem just about everywhere else, so might as well keep things consistent.

Reply:
Fixed